### PR TITLE
Add RuboCop to enforce style rules

### PR DESCRIPTION
**Pull request by sferik**
This is a follow-up to https://github.com/zdennis/activerecord-import/pull/250. It does a bit more cleanup and adds RubyCop to the test suite, so the build will fail there are any style guide regressions (as specified in `.rubocop.yml`).

It also adds the following two `rake` tasks:

```
rake rubocop                      # Run RuboCop
rake rubocop:auto_correct         # Auto-correct RuboCop offenses
```

The first task is the one that runs on Travis CI and will report any violations of the style guide. The second is intended to be run locally and will attempt to fix any style guide violations that can be automatically corrected (remove trailing whitespace, convert tabs into spaces, etc.)

If you’d like, I can also add a few sentences in the `README` or create a `CONTRIBUTING` doc that informs potential developers about the style guide and how to adhere to it.

This will ensure a consistent style is used throughout this project, which should make the code easier to work with going forward.

### CODE REVIEW

File: lib/activerecord-import/import.rb [hunk 1]
```diff
@@ -480,8 +480,8 @@ def import_without_validations_or_callbacks( column_names, array_of_attributes,
 
         # perform the inserts
         (number_inserted, ids) = connection.insert_many( [insert_sql, post_sql_statements].flatten,
-                                                  values_sql,
-                                                  "#{self.class.name} Create Many Without Validations Or Callbacks" )
+          values_sql,
+          "#{self.class.name} Create Many Without Validations Or Callbacks" )
```
Comment (zdennis) [inline lib/activerecord-import/import.rb]: :thumbsup: for `with_fixed_indentation` over `with_first_parameter`. Can we turn off `with_first_parameter` as a viable option altogether? It's an abomination to all code that hopes to good in this world. :)

**Comment by zdennis** [comment]
Look at all of the inconsistencies in this code base! Who's been maintaining this over the years?! 

Thank you @sferik for bringing order to the chaos and improving the state of things! It will help the code-base evolve over time and the addition of rubocop will help contributions reach a level of consistency not yet seen in its 10 year history.

There may be some aesthetic differences between myself and Rubocop defaults, but I do enjoy consistent code and having a system in place for helping produce consistent code as the code-base continues to evolve. If anything feels like we're harming the world more than helping we can change as we go. I'd rather not nitpick now with all of the goodness this PR brings in.

@sferik, are there are any other PRs you'd like to see merged before this or would you like this merged before any others?

**Comment by sferik** [comment]
> Thank you @sferik for bringing order to the chaos and improving the state of things! It will help the code-base evolve over time and the addition of rubocop will help contributions reach a level of consistency not yet seen in its 10 year history.

My pleasure. Thank you for creating and maintaining this great library.

> There may be some aesthetic differences between myself and Rubocop defaults, but I do enjoy consistent code and having a system in place for producing consistent code as the code-base evolves. If anything feels like we're harming the world more than helping we can change as we go. I'd rather not nitpick now with all of the goodness this PR brings in.

Please feel free to change the defaults as you see fit. Many can be auto-corrected to whatever style is specified in `.rubocop.yml`.

> @sferik, are there are any other PRs you'd like to see merged before this or would you like this merged before any others?

That’s really up to you! I just rebased, so this branch currently merges cleanly into `master`. Doing so may cause conflicts in some other pull requests, so they will need to be rebased before they can be merged. If you’d like to merge any other conflicting pull requests before this one, I can always rebase again.

**Comment by sferik** [comment]
Rebasing this branch now.

**Comment by zdennis** [comment]
Thanks @sferik !

**Pull request merged**